apps/web/app/api/defaultResponderForAppDir.test.ts

```diff
@@ -33,7 +34,7 @@ describe("defaultResponderForAppDir", () => {
   it("should return an empty JSON response when handler resolves with no result", async () => {
     const f = vi.fn().mockResolvedValue(null);
     const req = { method: "GET", url: "/api/test" } as unknown as NextRequest;
-    const params = {};
+    const params = new Promise<Params>((resolve) => resolve({}));
```

Comment:
just fyi, `const params = Promise.resolve<Params>({})` is possible as well

Reply:
Nice! addressed!